depthai_examples/ros1_src/depth_crop_control.cpp

```diff
@@ -69,11 +72,34 @@ int main() {
     dai::Point2f topLeft(0.2, 0.2);
     dai::Point2f bottomRight(0.8, 0.8);
 
+    dai::node::MonoCamera::Properties::SensorResolution _monoResolution; 
```

Comment:
any specific reason for prefix `_`

Reply:
monoResolution was used as a parameter, "resolution" was too general, we needed another name, I added prefix '_' since is usually used to denote when a variable is used locally in a class (instead of this->variable_name)